src/pages/Explorer/Layouts/SocialTrend.svelte

```diff
@@ -1,21 +1,32 @@
 <script>
-  export let item
+  import { history } from '../../../redux'
+
+  export let item = {}
+
+  const PER_ROW = 4
+
+  $: ({ word, tags, volume } = item)
+  $: TOTAL_TAGS = tags.length
```

Comment:
Uppercase is used for static constant variables, meanwhile this is the reactive local variable. Camel case should be used instead